src/averageAges.js

```diff
@@ -9,17 +9,53 @@
  * To calculate century:
  * Divide year of person's death by 100: Math.ceil(person.died / 100)
  *
+ * write code here
+ * avoid using loop and forEach
+ * replace `if ()` statement with &&, || or ?:
+ * without nesting
+ *
  * @param {object[]} people
  * @param {number} century - optional
  *
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const mans = people.reduce((allMans, man) => {
+    man.sex === 'm' && allMans.push(man);
```

Comment:
'm' is a magic string here and it's an antipattern. Consider using constants for this with a meaningful name